DEVELOPER_GUIDE.md

```diff
@@ -164,19 +143,20 @@ To **run OpenSearch from a compiled binary**, follow these steps:
 
 - Start a separate terminal and navigate to the directory where OpenSearch has been cloned using `cd OpenSearch`.
 - Run `./gradlew assemble` to create a local distribution.
-- Navigate to the project root directory (for example, `cd distribution/archives/linux-tar/build/install/opensearch-3.0.0-SNAPSHOT/`). Note: On macOS, replace `linux-tar` with `darwin-tar`.
-- Check whether the `extensions` directory exists in OpenSearch using `ls`.
-- If the directory does not exist, create it using `mkdir extensions`.
-- Navigate to the extensions folder using `cd extensions`.
-- Manually create a file titled `extensions.yml` within the extensions directory using an IDE or an inline text editor.
-- Return to the OpenSearch directory using `cd ..`.
 - Start OpenSearch using `./bin/opensearch`.
+- Hit the below sample REST API to initialize an extension
+```
+ curl -XPOST "localhost:9200/_extensions/initialize" -H "Content-Type:application/json" --data '{"name":"ad-extension","uniqueId":"ad-extension","hostAddress":"127.0.0.1","port":"4532","version":"1.0","opensearchVersion":"3.0.0","minimumCompatibleVersion":"3.0.0","dependencies":[{"uniqueId":"test1","version":"2.0.0"},{"uniqueId":"test2","version":"3.0.0"}]}'
```

Comment:
This scrolls off the screen in markdown.  

![Screenshot 2023-06-12 at 2 24 21 PM](https://github.com/opensearch-project/opensearch-sdk-java/assets/9291703/0bfd1aea-2ffb-459d-815d-08b536c21f40)

While there's a copy/paste button it would be more useful if it all fit on one page.  You can use the backslash `\` for bash commands to continue on the next line, so I think this might be better as:
```bash
 curl -XPOST "localhost:9200/_extensions/initialize" -H "Content-Type:application/json" --data '{ \
"name":"ad-extension", \
"uniqueId":"ad-extension", \
"hostAddress":"127.0.0.1", \
"port":"4532", \
"version":"1.0", \
"opensearchVersion":"3.0.0", \
"minimumCompatibleVersion":"3.0.0", \
"dependencies":[{"uniqueId":"test1","version":"2.0.0"},{"uniqueId":"test2","version":"3.0.0"}] \
}'
```